src/oclHashcat.c

```diff
@@ -384,7 +384,7 @@ const char *USAGE_BIG[] =
   "       --bitmap-max=NUM              Maximum number of bits allowed for bitmaps",
   "       --cpu-affinity=STR            Locks to CPU devices, seperate with comma",
   "  -d,  --opencl-devices=STR          OpenCL devices to use, separate with comma",
-  "       --opencl-platform=STR         OpenCL platform to use, in case multiple OpenCL platforms are present",
+  "       --opencl-platform=NUM         OpenCL platform to use, in case multiple OpenCL platforms are present",
   "  -w,  --workload-profile=NUM        Enable a specific workload profile, see references below",
```

Comment:
Perhaps you should also shorten it to *OpenCL platform to use, in case multiple platforms are present*